model-optimizer/mo/main.py

```diff
@@ -100,13 +100,13 @@ def prepare_ir(argv: argparse.Namespace):
     is_tf, is_caffe, is_mxnet, is_kaldi, is_onnx = deduce_framework_by_namespace(argv)
 
     fem = argv.feManager
-    new_front_ends = []
+    ngraph_front_ends = []
```

Comment:
I don't see any valid reason for this renaming. We should avoid usage of all internal names in the code when possible.

Reply:
How long will "new" have any meaning?